migtests/tests/import-file/run-import-file-test

```diff
@@ -31,6 +31,10 @@ main() {
 	run_ysql yugabyte "DROP DATABASE IF EXISTS ${TARGET_DB_NAME};"
 	run_ysql yugabyte "CREATE DATABASE ${TARGET_DB_NAME}"
 
+	# create the credentials file for the authentication for GCS bucket
+	step "Create the credentials file for GCS default authentications"
+	${TEST_DIR}/create_gcs_credentials_file
```

Comment:
This should happen from the workflow definition file.

Note that we want `run-import-file-test` to be executable directly from dev server.